### DISCUSSION TITLE
Biofuel and Oil textures inside of buckets are higher resolution and flowing downwards

### DISCUSSION THREAD

**Discussion by saltyseadoggo** [TechReborn/TechReborn#2528] [Ideas]
**Describe the bug**
Whilst the bucket item is a 16x16 sprite, the biofuel and oil textures rendered inside of them are 32x32. This is a texturing "don't" known as 'mixels', and it makes these two buckets look strange.
Furthermore, the fluid textures are animated to appear to be flowing downward inside of their buckets, which does not make sense.

**Steps to Reproduce**
- Create a new world in Creative mode.
- Obtain either the Biofuel item or the Oil item. Observe the item texture.

**Screenshots**
Note that the fluids are higher resolution than the buckets. The animation can't be shown in a humble screenshot.
![2020-12-21_20 32 23](https://user-images.githubusercontent.com/69029346/102849275-a9bba080-43cb-11eb-884d-96435acbeedf.png)

**Environment (please complete the following information with the version):**
 - Minecraft: 1.16
 - Mod Loader: Fabric

**Comment by jusvit** [comment]
Eh, people don't excessively look at bucket on frames and the flowing down due to the rotation of the png, I'll fix this but as for the pixel size, it's fine.

**Reply by saltyseadoggo** [reply]
The pixel size is also visible in inventories. Imo, it looks pretty clashy with everything else in the game, including the bucket sprite itself
